blog-investigation/src/blog/components/BlogCard.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Card } from "antd";
+
+import PrimaryBtn from "../../shared/components/buttons/PrimaryBtn";
+import Translator from "../../shared/components/Translator";
+import DisabledBtn from "../../shared/components/buttons/DisabledBtn";
+
+function BlogCard(props) {
+  return (
+    <Card title={<Translator text={props.name} />} className="blog-card">
+      <p>{<Translator text={props.description} />}</p>
+      <span className="blog-card-content">
+        <p>
```

Comment:
not valid html: block element inside the inline element. please replace span with block element